tests/foreman/sys/test_hot_backup.py

```diff
@@ -0,0 +1,156 @@
+# -*- encoding: utf-8 -*-
+"""Test class for ``katello-backup``
+
+@Requirement: HotBackup
+
+@CaseAutomation: notautomated
+
+@CaseLevel: Acceptance
+
+@CaseComponent: Backup
+
+@TestType: System
+
+@CaseImportance: High
+
+@Upstream: No
+"""
+
+from robottelo.decorators import stubbed
+from robottelo.test import TestCase
+
+
+class HotBackupTestCase(TestCase):
+    """Implements ``katello-backup --online`` tests"""
+
+    @classmethod
+    def setUpClass(cls):
+        super(HotBackupTestCase, cls).setUpClass()
+
+    @stubbed()
```

Comment:
@tier3 - I think  all stubs related to backup/restore should fall into tier3 tests. 

- if you'r updating `tier3` decorator then please add another tag in docstrings like ` @CaseLevel: System`
This comment is applicable for all tier3 tests.